doc/Command-Reference.md

```diff
@@ -11997,13 +11997,254 @@ This command displays <vlan_name> configuration.
 - Example:
 
   ```
-  admin@sonic:~$ show vxlan name tunnel3
-  vxlan tunnel name    source ip    destination ip    tunnel map name    tunnel map mapping(vni -> vlan)
-  -------------------  -----------  ----------------  -----------------  ---------------------------------
-  tunnel3              10.10.10.10  30.10.10.10       tmap2              1235 -> 200
+  admin@sonic:~$ show vxlan name vtep
+  vxlan tunnel name    source ip    destination ip    tunnel map name     tunnel map mapping(vni -> vlan)
+  -------------------  -----------  ----------------  ------------------  ---------------------------------
+  vtep                 1.1.1.1                        map_3000_Vlan30     3000 -> Vlan30
   ```
 
-Go Back To [Beginning of the document](#) or [Beginning of this section](#vxlan--vnet)
+**show vxlan interface**
+
+This command displays brief VTEP information.
+
+- Usage:
+
+  ```
+  show vxlan interface
+  ```
+
+- Example:
+
+  ```
+  admin@sonic:~$ show vxlan interface
+  VTEP Information:
+
+          VTEP Name : vtep, SIP  : 1.1.1.1
+          NVO Name  : nvo,  VTEP : vtep
+          Source interface  : Loopback0
+  ```
+
+**show vxlan vlanvnimap**
+
+This command displays all the VLAN VNI mappings.
+
+- Usage:
+
+  ```
+  show vxlan vlanvnimap
+  ```
+
+- Example:
+
+  ```
+    admin@sonic:~$ show vxlan vlanvnimap
+    +--------+-------+
+    | VLAN   | VNI   |
+    +========+=======+
+    | Vlan30 | 3000  |
+    +--------+-------+
+    Total count : 1
+  ```
+
+**show vxlan vrfvnimap**
+
+This command displays all the VRF VNI mappings.
+
+- Usage:
+
+  ```
+  show vxlan vrfvnimap
+  ```
+
+- Example:
+
+  ```
+  admin@sonic:~$ show vxlan vrfvnimap
+  +-------+-------+
+  | VRF   |   VNI |
+  +=======+=======+
+  | Vrf-1 |   104 |
+  +-------+-------+
+  Total count : 1
+  ```
+
+**show vxlan remote_mac <remoteip/all>**
+
+This command displays all MACs learnt from a specified remote VTEP IP, or from all of the remote VTEP IPs for all VNIs.
+
+- Usage:
+
+  ```
+  show vxlan remote_mac <remoteip/all>
```

Comment:
>remote_mac [](http://example.com/codeflow?start=13&length=10)

same here

Reply:
Correct parameter name to be 'remotemac'